.github/workflows/generic-test-runner.yml

```diff
@@ -2,31 +2,87 @@ name: Generic Test Runner
 on:
   workflow_dispatch:
     inputs:
-      directtory:
+      network:
+        description: 'Network to run tests against'
+        required: true
+        default: 'SIMULATED'
+      wsURL:
+        description: 'Chain WS URL (Skip with SIMULATED)'
+        required: false
+      httpURL:
+        description: 'Chain HTTP URL (Skip with SIMULATED)'
+        required: false
+      fundingKey:
+        description: 'Private key to fund test (Skip with SIMULATED)'
+        required: false
+      directory:
         description: 'Directory to run tests from'
         required: true
         default: 'smoke'
       test:
         description: 'Test to run'
         required: true
-        default: 'all'
-      test-inputs:
+        default: 'OCRBasic'
+      testInputs:
         description: 'Custom test inputs'
         required: false
         default: ''
 env:
   ENV_JOB_IMAGE: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com/chainlink-tests:${{ github.sha }}
   CHAINLINK_IMAGE: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com/chainlink
```

Comment:
@kalverra Is it possible to run tests with a release version? I ask because that is sometimes the requirement in some tests.

Reply:
Yes, it's possible with any tag or branch